docs/structuring.md

```diff
@@ -146,6 +146,33 @@ These generic types are composable with all other converters.
 ['1', None, '3']
 ```
 
+### Deques
+
+Deques can be produced from any iterable object. Types converting
+to deques are:
+
+- `Deque[T]`
+- `deque[T]`
+
+In all cases, a new **unbounded** deque (`maxlen=None`) will be returned, so this operation can be used to
+copy an iterable into a deque.
+If you want to convert into bounded `deque`, registering a custom structuring hook is a good approach.
+
+```{doctest}
+>>> cattrs.structure((1, 2, 3), deque[int])
+[1, 2, 3]
```

Comment:
This should probably be `deque([1, 2, 3])`?

Reply:
No? I think here is no limit.  If it's a sequence, whatever it actually is, it will be converted into `deque`.

Reply:
What I'm saying is, the result of the call will be `deque([1, 2, 3])`:

```python
>>> cattrs.structure((1, 2, 3), deque[int])
deque([1, 2, 3])
```
The example shows the result of the structure is just `[1, 2, 3]`.

Reply:
Oh. You're right! 😅 My bad